beaker-vue/src/components/panels/MediaPanel.vue

```diff
@@ -0,0 +1,161 @@
+<template>
+    <div class="media-focus">
+        <Toolbar class="media-toolbar">
+            <template #start>
+                <Button 
+                    icon="pi pi-arrow-left" 
+                    class="media-toolbar-button" 
+                    severity="primary" 
+                    outlined
+                    :onclick="() => {
+                        currentOutputOneIndexed -= (currentOutputOneIndexed - 1) < 1 
+                            ? 0 
+                            : 1
+                    }" 
+                />
+                <Button 
+                    icon="pi pi-arrow-right" 
+                    class="media-toolbar-button" 
+                    severity="primary"
+                    outlined
+                    :onclick="() => {
+                        currentOutputOneIndexed += (currentOutputOneIndexed + 1) > notebookOutputs.length 
+                            ? 0
+                            : 1
+                    }" 
+                />
+            </template>
+            <template #end>
+                <InputGroup>
+                    <InputGroupAddon class="media-dropdown-icon">
+                        <i class="pi pi-chart-bar"></i>
+                    </InputGroupAddon>
+                    <Dropdown 
+                        v-model="currentOutputOneIndexed"
+                        :options="[...Array(notebookOutputs.length + 1).keys()].slice(1)"
+                    />
+
+                    <InputGroupAddon>/ {{ notebookOutputs.length ?? 0 }}</InputGroupAddon>
+                </InputGroup>
+            </template>
+        </Toolbar>
+        <div class="media-mime-bundle">
+            <div 
+                v-html="currentMimeBundle['text/html']" 
+                v-if="currentMimeBundle && currentMimeBundle['text/html'] !== undefined"
+                class="code-cell-output"
+                
+            />
+            <BeakerMimeBundle 
+                :mime-bundle="currentMimeBundle" 
+                v-else-if="currentMimeBundle !== undefined"
+            />
```

Comment:
Why do we need to have a conditional here if there is html? 
What if there is both an html and an image, why should it be one or the other?